content/blog/okta-sso-integration-for-green-lake-and-aruba-central.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Okta SSO integration for Green Lake and Aruba Central
+date: 2023-02-14T20:11:58.846Z
+externalLink: https://blog.wifig-guys.com
```

Comment:
Hi Will,

Not sure if you want to create a "blog link" that points back to your external and existing blog post in the web site https://blog.wifi-guys,com or if you would like to create a new blog post submission to be published on the HPE Developer Community.

If it is the later, no need to specify the external link in your entry. 
Now, if you want to create a blog link that points the readers back to your original post in your web site, just keep the external link (please make sure to adjust the website URL, there is a typo in it) and there is no need to fill in the "Body". Please refer to guidelines here: 
https://github.com/hpe-dev-incubator/hpe-dev-portal/blob/master/docs/ContributorGuide-v2.md#onboarding-an-external-blog-post